tests/utils/test_train_utils.py

```diff
@@ -26,3 +33,31 @@ def test_align_token_features():
     assert np.all(actual_features[0][3] == np.mean(token_features[0][3:5], axis=0))
     # embedding is split into 4 sub-tokens
     assert np.all(actual_features[0][4] == np.mean(token_features[0][5:10], axis=0))
+
+
+@pytest.mark.parametrize(
+    "split_entities_config, expected_initialized_config",
+    [
+        (
+            SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE,
+            {SPLIT_ENTITIES_BY_COMMA: SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE},
+        ),
+        (
+            {"address": False, "ingredients": True},
+            {
+                "address": False,
+                "ingredients": True,
+                SPLIT_ENTITIES_BY_COMMA: SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE,
+            },
+        ),
+    ],
+)
+def test_init_split_entities_config(
```

Comment:
Is there a small real-life use-case for the functionality with `TEDPolicy` that we could add to the tests?

Reply:
I think the functionality should be covered by the existing test -- there shouldn't be a difference between the behaviour for `TEDPolicy` or one of the `EntityExtractors`, because we'd use the split configs only in e2e, when `TEDPolicy` extracts entities

Reply:
OK - great!